src/store/reducer/RootReducer.ts

```diff
@@ -0,0 +1,60 @@
+import {createSlice, createAsyncThunk} from '@reduxjs/toolkit';
+import {Crypto} from 'interfaces';
+
+const initialState = {
+  cryptos: [] as Crypto[],
+  loading: false,
+};
+
+export const getCrypto = createAsyncThunk(
+  'crypto/getCrypto',
+  async (name: string) => {
+    try {
+      const {data} = await fetch(
+        `${process.env.REACT_APP_BASE_URL}${name}/metrics`,
+      ).then(res => res.json());
+      return data;
+    } catch (error) {
+      return error;
+    }
+  },
+);
+
+export const updateCrypto = createAsyncThunk(
```

Comment:
This is outside the scope, It should be implemented in the last task, remove it from this PR and use it when is requested